bigbluebutton-html5/imports/ui/components/presentation/component.jsx

```diff
@@ -850,6 +854,7 @@ class Presentation extends PureComponent {
                 : null,
           }}
         >
+          <h2 class="sr-only">{intl.formatMessage(intlMessages.presentationHeader)}</h2>
```

Comment:
```suggestion
          <h2 className="sr-only">{intl.formatMessage(intlMessages.presentationHeader)}</h2>
```

based on ESLint complaints

> /home/runner/work/bigbluebutton/bigbluebutton/bigbluebutton-html5/imports/ui/components/webcam/component.tsx
>   283:19  error  Unknown property 'class' found, use 'className' instead  react/no-unknown-property